src/index.js

```diff
@@ -3,11 +3,13 @@ import * as generateBookmarks from "./generate-bookmarks.js";
 import * as saveBookmarks from "./save-bookmarks.js";
 import * as mergeBookmarks from "./merge-bookmarks.js";
 import * as convertBookmarks from "./convert-bookmarks.js";
+import * as checkBookmarks from "./check-bookmarks.js";
```

Comment:
I think we should also add this to the developer ready my to explain how the functions can be used to be consistent with other modules. Its just a quick note for people who might want to import into other projects

Reply:
Added documentation.